pkg/kubectl/resize.go

```diff
@@ -150,15 +151,15 @@ func (resizer *ReplicationControllerResizer) Resize(namespace, name string, newS
 	}
 	if retry == nil {
 		// Make it try only once, immediately
-		retry = &RetryParams{interval: time.Millisecond, timeout: time.Millisecond}
+		retry = &RetryParams{time.Millisecond, time.Millisecond}
```

Comment:
(personal preference) Recommend leaving the field names in -- makes adding struct members less painful. May not matter here.